### ISSUE TITLE
Redlib not working anymore on my end

### ISSUE THREAD

**Issue by aeon3** [redlib-org/redlib#345]
It's been a while I haven't been able to use redlib due to this message.

I'm using a VPN and brave browser or librewolf.

Sometimes it would randomly work but as of last few weeks, completely dead for me.

Any ideas?

```
Failed to parse page JSON data: expected value at line 1 column 1 | /r/popular/hot.json?&raw_json=1&geo_filter=GLOBAL
[Reddit Status](https://www.redditstatus.com/)

Expected something to work? [Report an issue](https://github.com/redlib-org/redlib/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=%F0%9F%90%9B+Bug+Report%3A+Failed%20to%20parse%20page%20JSON%20data:%20expected%20value%20at%20line%201%20column%201%20|%20/r/popular/hot.json?&raw_json=1&geo_filter=GLOBAL)

Head back [home](https://libreddit.privacydev.net/)?
```

**Comment by unclearParadigm** [comment]
Reddit is carrying out IP bans again. This brought down many public instances today.

**Comment by sigaloid** [comment]
Yep. There's not a ton I can do in this case - if they ban an IP, it's on all of their CDN endpoints. In addition I'd be willing to bet they're either doing it manually by requesting some canary post ID, or based off of traffic behavior. Not some overarching detection of the software or else every individual instance would be banned.

There's been some solutions dropped, including routing over Cloudflare Warp in the docker compose: https://github.com/redlib-org/redlib/issues/318#issuecomment-2490882093

**Comment by ButteredCats** [comment]
Looks like they changed something based off how many instances got nuked the other day https://github.com/redlib-org/redlib-instances/commit/392f910a1c9cb3bde38dc8b4071e303c466aef62

I set up Cloudflare warp on my containers yesterday morning and today my IP is already unbanned, so seems automated.
